conda/recipes/cudf/meta.yaml

```diff
@@ -67,7 +70,6 @@ requirements:
     {% if cuda_major == "11" %}
     - cudatoolkit
     {% else %}
-    - cuda-cudart-dev
     - cuda-nvrtc
     - libcufile-dev  # [linux64]
```

Comment:
If we made the change above, `cuda-cudart-dev` could stay just like the other dev packages

This would also help inform us that `cuda-cudart` is dynamically linked to (as opposed to statically linked to). In the static linking case, we wouldn't list `cuda-cudart-dev` here (even ignoring the requirement constraint)

Reply:
I don't have strong feelings here. I decided that it was okay to remove `cuda-cudart-dev` because we had that package in `host` _only_ as a means of getting the run-export of `cuda-cudart`. We don't need to install it into `host` for the build to work. I thought it was clearer to remove that package and just have the run dependency on `cuda-cudart`. If you like it the other way, I can do that.

Reply:
At first I thought it would be reasonable to treat cudart differently than math libraries (maybe only math libraries need `-dev` packages. But maybe you're right and I should do the same thing for cudart. Take a peek at cuml/cugraph and see what you think of those recipes. I'll try to align things.

Reply:
Are cuML and cuGraph statically linking to `cudart`? If so, what they are doing seems reasonable

Reply:
I just pushed 8090e502bf3680142d68898036a2efe28d82da33. @jakirkham Can you re-review and see if this is what you're seeking?

> Are cuML and cuGraph statically linking to `cudart`?

I don't know for sure, but I think most RAPIDS conda packages are dynamically linked to cudart. I could be wrong.

Reply:
Thanks Bradley! 🙏 LGTM ✅ 

As to cuML, think you are right. Noticed something odd in the cuML CI logs. Filed issue: https://github.com/rapidsai/cuml/issues/5765